WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -696,66 +683,10 @@ private void stopStatsService() {
         }
     }
 
-    /*
-     * receiver for broadcast from StatsService which alerts when stats update has started/ended
-     */
-    private final BroadcastReceiver mReceiver = new BroadcastReceiver() {
-        @Override
-        public void onReceive(Context context, Intent intent) {
-            String action = StringUtils.notNullStr(intent.getAction());
-
-            if (action.equals(StatsService.ACTION_STATS_UPDATING)) {
```

Comment:
This code was doing:
* `mSwipeToRefreshHelper.setRefreshing(mIsUpdatingStats);` (still there, line 688)
* and login error checking. But since this event has been introduced: https://github.com/wordpress-mobile/WordPress-Android/blob/0660f184fb71c2ad0ab5f440a4d419dbe4a3cccd/WordPress/src/main/java/org/wordpress/android/WordPress.java#L267-L274 this code was never called (actually EXTRA_IS_ERROR was never set).

So I removed login error checking and only kept the call to `mSwipeToRefreshHelper.setRefreshing`. @daniloercoli can confirm.